app/controllers/api/v2/remote_execution_features_controller.rb

```diff
@@ -29,6 +29,14 @@ def update
         process_response @remote_execution_feature.update(remote_execution_feature_params)
       end
 
+      api :GET, '/api/hosts/:id/available_remote_execution_features', N_('List available remote execution features for a host')
+      param :id, :identifier, :required => true
+      def available_remote_execution_features
+        host = Host.find(params[:id])
+        @remote_execution_features = resource_scope
+        @permissions = {:can_run_job => (authorized_for(controller: :job_invocations, action: :create) && (!host.infrastructure_host? || User.current.can?(:execute_jobs_on_infrastructure_hosts))) }
```

Comment:
I was thinking about not exposing the permission directly and just returning an empty array in case the user doesn't have the necessary permissions, but this works too

Reply:
It doesn't catch the case when for some reason the user doesnt have any REX features but has permissions to run jobs, if thats possible

Reply:
It shouldn't happen, but it is theoretically possible. Nevermind then